Home/Home.html

```diff
@@ -1,13 +1,165 @@
+            <div class="part2">@National Museum of korea</div>
+        </div>
+    </section>
+    <section class="MainProgram">
+        <div class="title1">
+            Main Programs
+        </div>
+        <hr>
+        <div class="programs">
+            <div class="Program">
+                <img src="/images/fg.png" alt="">
+                <p class="classtitle-1">
+                    Featured Games
+                </p>
+                <p class="desc-3">
+                    Highlight popular or trending games that are generating a lot of interest.
+                </p>
+            </div>
+            <div class="Program">
+                <img src="/images/wt.jpg" alt="">
+                <p class="classtitle-1">
+                    Latest Walkthroughs
+                </p>
+                <p class="desc-3">
+                    Showcase recently published walkthroughs for newly released or popular games.
+                </p>
+            </div>
+            <div class="Program">
+                <img src="/images/gr.jpeg" alt="">
+                <p class="classtitle-1">
+                    Game Reviews
+                </p>
+                <p class="desc-3">
+                    Provide summaries and ratings for games, helping users make informed decisions.
+                </p>
+            </div>
+            <div class="Program">
+                <img src="/images/gc.png" alt="">
+                <p class="classtitle-1">
+                    Game Categories
+                </p>
+                <p class="desc-3">
+                    Organize games into genres or categories (e.g., action, adventure, puzzle) for easy browsing
+                </p>
+            </div>
+            <div class="Program">
+                <img src="/images/spotlight_3174373.png" alt="">
+                <p class="classtitle-1">
+                    Community Spotlights
+                </p>
+                <p class="desc-3">
+                    Showcase user-generated content such as fan art, cosplay, or impressive gameplay videos.
+                </p>
+            </div>
+        </div>
+
+        <button id="Jp">Join the program</button>
+    </section>
+    <section class="Sponsors">
+        <p class="title1">
+            Sponsored By
+        </p>
+        <div class="projects">
+            <div class="r1">
+             </div>
+            <div class="hidden" id="hiddenDiv">
+                
+            </div>
+        </div>
+
+
+        <button class="more" onclick="toggleDiv()">More<img class="icon" src="/images/down.png" alt=""></button>
+    </section>
+    <section class="partners">
+        <p class="desc8">Partners</p>
+        <hr>
+        <div><img src="/images/nvidia.png" alt="">
+            <img src="/images/ubisoft.png" alt="">
+            <img src="/images/intel.png" alt="">
+            <img src="/images/logitech.png" alt="">
+        </div>
```

Comment:
- [ ] Please ensure to provide the alt attribute when using images throughout your application. It is important to use this so that whenever an image is not displayed on the browser the user has an idea of what images was supposed to load and this is also useful for screen readers and assistive technologies 👍